IS0501Test.py

```diff
@@ -802,6 +803,40 @@ def test_40(self, test):
             else:
                 return test.UNCLEAR("Not tested. No resources found.")
 
+    def test_41(self, test):
+        """SDP transport files pass SDPoker tests"""
+
+        api = self.apis[CONN_API_KEY]
+        rtp_senders = []
+        if self.is05_utils.compare_api_version(api["version"], "v1.1") >= 0:
+            # Find all RTP senders for v1.1+
+            for sender in self.senders:
+                url = "single/senders/{}/transporttype".format(sender)
+                valid, response = self.is05_utils.checkCleanRequestJSON("GET", url)
+                if valid:
+                    if response == "urn:x-nmos:transport:rtp":

```

Comment:
Will need to be adjusted based on final conclusion of https://github.com/AMWA-TV/nmos-device-connection-management/issues/57